torchinductor/utils.py

```diff
@@ -9,6 +9,18 @@
 from torch.fx.immutable_collections import immutable_dict
 from torch.fx.immutable_collections import immutable_list
 
+from torchinductor import config
+
+
+def init_opt_level():
```

Comment:
Perhaps we should have `torchinductor.set_optimization_level(0)` instead of `opt_level` as a config.

If the user does:
```
torchinductor.config.opt_level = 0
torchinductor.config.cudagrapha = True
```
They will not get the result they expect.